decidim-core/app/packs/src/decidim/sw/sw.js

```diff
@@ -0,0 +1,32 @@
+import {
+  pageCache,
+  imageCache,
+  staticResourceCache,
+  offlineFallback
+} from "workbox-recipes";
+import { setDefaultHandler } from "workbox-routing";
+import { NetworkOnly } from "workbox-strategies";
+
+/**
+ * This is a workaround to bypass a webpack compilation error
+ *
+ * The InjectManifest function requires the __WB_MANIFEST somewhere in this file,
+ * however, we cannot add precacheAndRoute as the issue suggests,
+ * as the other workbox-recipes won't work properly
+ *
+ * See more: https://github.com/GoogleChrome/workbox/issues/2519#issuecomment-634164566
+ */
+// eslint-disable-next-line no-unused-vars
+const dummy = self.__WB_MANIFEST;
+
+pageCache()
+
+staticResourceCache();
+
+imageCache();
+
+setDefaultHandler(
+  new NetworkOnly()
+);
```

Comment:
Should you also cache the webmanifest? 

When I'm in offline mode the manifest is still required:

<img width="929" alt="Screenshot 2021-12-18 at 07 05 38" src="https://user-images.githubusercontent.com/17616/146631237-e02ef994-2508-43a2-8e82-6610bb545deb.png">

Reply:
I'm researching this thing, it's not just like that. If so, the workbox-recipes would have handled it already, it looks like something about the InjectManifest or any missing configuration. I'll keep digging on this.

Reply:
It's hard to get some point here. The current version of the webmanifest is so simple that it won't work any PWA features: a2hs, splash, icons or local installation.
However I tested that out in the webpush-test repo, from my mobile phone as Decidim as an installed app, and I cannot notice any issue when I got into offline.

Let's come back to this once the manifest will be more complex